src/main/java/appeng/fluids/container/slots/IFluidSlot.java

```diff
@@ -27,9 +27,9 @@
  * @version rv6 - 3/05/2018
  * @since rv6 3/05/2018
  */
-public interface ISlotFluid
+public interface IFluidSlot

```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Annotate the "IFluidSlot" interface with the @FunctionalInterface annotation [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1609)